modules/test/playwright/tests/frontend-ckeditor-sample-web/stylesCombo.spec.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import getPageDefinition from '../layout-content-page-editor-web/utils/getPageDefinition';
+import getWidgetDefinition from '../layout-content-page-editor-web/utils/getWidgetDefinition';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	isolatedSiteTest,
+	loginTest()
+);
+
+test('@LPD-33712 stylesCombo dropdown visible with maximized', async ({
+	apiHelpers,
+	page,
+	site,
+}) => {
+	let layout: Layout;
+
+	await test.step('Create a content site and the ckeditor sample widget', async () => {
+		const widgetDefinition = getWidgetDefinition({
+			id: getRandomString(),
+			widgetName:
+				'com_liferay_editor_ckeditor_sample_web_internal_portlet_CKEditorSamplePortlet',
+		});
+
+		layout = await apiHelpers.headlessDelivery.createSitePage({
+			pageDefinition: getPageDefinition([widgetDefinition]),
+			siteId: site.id,
+			title: getRandomString(),
+		});
+	});
+
+	await test.step('Select Maximized button and check stylesCombo dropdown', async () => {
+		await page.goto(
+			`${liferayConfig.environment.baseUrl}/web${site.friendlyUrlPath}${layout.friendlyUrlPath}`
+		);
+
+		const ckeditorSampleWidgetClassicTab = page
+			.getByRole('tab', {
+				name: 'Classic',
+			})
+			.first();
+
+		await ckeditorSampleWidgetClassicTab.waitFor({state: 'visible'});
+		await ckeditorSampleWidgetClassicTab.click();
+
+		const maximizedButton = page.getByLabel('Maximize').first();
```

Comment:
Why first? There should be only one `Maximize` tab. Maybe we can be more specific with something like
```js
.getByRole('button', {
    name: 'Maximize',
})
```